libraries/AP_RangeFinder/AP_RangeFinder_uLanding.h

```diff
@@ -0,0 +1,31 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+#pragma once
+
+#include "RangeFinder.h"
+#include "RangeFinder_Backend.h"
+
+#define ULANDING_COEFF 2.5
```

Comment:
note that we now pass a compiler flag to make float constants default to single precision, so we don't need to be careful any more.